main.tf

```diff
@@ -94,18 +96,23 @@ resource "helm_release" "castai_evictor" {
   namespace       = "castai-agent"
   create_namespace = true
   cleanup_on_fail = true
+  wait = true
 
   set {
     name  = "replicaCount"
     value = "0"
   }
 
   depends_on = [helm_release.castai_agent]
+
+  lifecycle {
+    ignore_changes = [set, version]
```

Comment:
discussed with Marcin a bit. there is `all` keyword we could use, but then we lose control to update name/repo in case of issues